contract_manager/src/executor.ts

```diff
@@ -1,7 +1,53 @@
 import { parseVaa } from "@certusone/wormhole-sdk";
-import { decodeGovernancePayload } from "xc_admin_common";
+import {
+  DataSource,
+  EvmExecute,
+  decodeGovernancePayload,
+} from "xc_admin_common";
 import { DefaultStore } from "./store";
-import { PrivateKey } from "./base";
+import { PrivateKey, TxResult } from "./base";
+import { EvmExecutorContract } from "./contracts";
+import { EvmChain } from "./chains";
+
+// TODO: A better place for this would be `base.ts`. That will require
+// significant refactor. Todo in separate PR.
+interface GovernanceContract {
```

Comment:
why not just use PriceFeedContract?

Reply:
The EvmExecutorContract doesn't and shouldn't implement PriceFeedContract.